README.md

```diff
@@ -23,6 +23,19 @@ Finally, update cypress/support/index.js file to include cypress-pact commands v
 import 'package-name'
 ```
 
+## Configuration
+By default, this plugin omits most cypress auto-generated HTTP headers. 
+To exclude more customised headers in your pact, add allowed headers as a list of string in `cypress.json` under key `env.headersBlocklist`. Eg. in your `cypress.json`
```

Comment:
```suggestion
To exclude other headers in your pact, add them as a list of strings in `cypress.json` under key `env.headersBlocklist`. Eg. in your `cypress.json`
```

Reply:
👍🏾 

suggestion - add them as an array of strings